public/app/plugins/datasource/azuremonitor/plugin.json

```diff
@@ -111,7 +111,9 @@
     },
     "links": [
       { "name": "Learn more", "url": "https://grafana.com/docs/grafana/latest/datasources/azuremonitor/" },
-      { "name": "Apache License", "url": "https://github.com/grafana/azure-monitor-datasource/blob/master/LICENSE" }
+      { "name": "Apache License", "url": "https://github.com/grafana/azure-monitor-datasource/blob/master/LICENSE" },
```

Comment:
@Ukochka I'm not sure this is correct given its an archived repo

Reply:
yeah it was already broken before, there is no change in this PR to this link.

But it can be improved - probably better to link towards https://github.com/grafana/grafana/blob/HEAD/LICENSE

Reply:
> yeah it was already broken before

in my haste i'm missing where it was just a comma being added, read it as the full line